crates/query-engine/translation/tests/goldenfiles/aggregate_limit_offset_order_by/configuration.json

```diff
@@ -1,5 +1,19 @@
 {
-  "version": "4",
+  "version": "5",
```

Comment:
Looks like you're bumping all the test configuration versions too? Perhaps do that in a separate PR too? It's hard to know if any changes in here are meaningful or not.

Reply:
agreed, will put that with the one adding the schemas